html/whatIsCodeGenerator.html

```diff
@@ -56,6 +56,12 @@ <h2 id="configurePlugin">Configuration Parameters for Customizing the Code Gener
       Each domain is represented by a directory under <code>src/main/resources/GUIData</code>.
       <pre data-src="../sample-files/whatIsCodeGenerator-sample5.xml" data-line="15-18" class="line-numbers"></pre>
     </li>
+    <li><code>&lt;customElements /&gt;</code> - This configuration parameter represents the list of custom elements to be registered to the code generator.
+      Each custom element should extend the <code>AbstractElement</code> be sure to implement all constructors.
+      <pre data-src="../sample-files/whatIsCodeGenerator-sample6.xml" data-line="15-18" class="line-numbers"></pre>
+      Custom element sample.
+      <pre data-src="../sample-files/whatIsCodeGenerator-sample-element.java" class="line-numbers"></pre>
```

Comment:
don't really need the line numbers unless you want to highlight them. you can omit the `class=` definitioin